start-ecs

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# Start script for register-api that assumes environment variables are already provided
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PORT=${PORT:=10000}
```

Comment:
Can we take this default port out too. We've had issues with this in the past

Reply:
Removed default port and now setting in ECS task definition.